### DISCUSSION TITLE
Custom frame rate on looping mode

### DISCUSSION THREAD

**Discussion by joeybassraft** [gopro/labs#1077] [Q&A]
Is there a way to record 1080p 30fps in looping mode? I have a hero 11
I know you can record at 4K 30fps in looping mode but I don't need the quality as much as the battery life/overheating

Thanks

**Comment by dnewman-gpsw** [comment]
No, just 1080p60 in the lowest.

**Reply by joeybassraft** [reply]
Okay, thanks for letting me know!

**Reply by dnewman-gpsw** [reply]
While not Loop mode, Labs could have a solution.  

Set 1080p30 in video mode, not loop.

Scan:  ```!C16!8N!R```  (or make this a boot command)

This will clear your media so that 16 GB is free, checking every 8 seconds.  The oldest media will be removed first.  So this operated like a loop mode.  With this script the camera can capture continuously.

**Comment by joeybassraft** [comment]
Thanks for looking into this, unfortunately I don't think it will work for
me because I want to save several short clips (I use the 5min setting on
looping) I might switch to continuous recording instead and cut it up in
post.

**Reply by dnewman-gpsw** [reply]
You can set  custom chapter sizes for a similar result

**Reply by dnewman-gpsw** [reply]
It maybe not be viable. I'm seeing some issues with continuous recording this way.
